packages/markdown/__tests__/variable-parser.test.js

```diff
@@ -1,6 +1,6 @@
 const unified = require('unified');
 const remarkParse = require('remark-parse');
-const parser = require('../variable-parser');
+const parser = require('../processor/parse/variable-parser');
```

Comment:
This reminds me... we still have to be able to insert variables inside of our dash editor. Try typing << in the current dash editor and you'll see what I mean!